src/terminal/adapter/terminalOutput.cpp

```diff
@@ -110,17 +140,18 @@ void TerminalOutput::EnableGrTranslation(boolean enabled)
 wchar_t TerminalOutput::TranslateKey(const wchar_t wch) const noexcept
 {
     auto wchFound = wch;
-    if (!_ssTranslationTable.empty())
+    if (_ssSetNumber >= 2 && _ssSetNumber <= 3)

```

Comment:
(Is this a complicated way to say `is 2 or 3`?)

Reply:
Yeah. I don't know that I was thinking. Will change that.